build.py

```diff
@@ -907,6 +904,10 @@ def configure(conf, env):
     if not env['NINJA']:
         env['NINJA'] = where_is(env, 'ninja-build') # Fedora...
 
+    if not env.has_key('_LIBDEPS_GET_LIBS'):
```

Comment:
Doesn't this just error hard on old branches even with --link-model=object? I think you want to ` and env['_LIBDEPS'] != '$_LIBDEPS_OBJS'`